README.md

```diff
@@ -201,6 +201,7 @@ separately from clients created with `new appInsights.TelemetryClient()`.
 | distributedTracingMode          | Sets the distributed tracing modes (Default=AI)                                                            |
 | enableAutoCollectExternalLoggers| Sets the state of console. If true logger activity will be sent to Application Insights |
 | enableAutoCollectConsole        | Sets the state of logger tracking (enabled by default for third-party loggers only). If true, logger auto collection will include console.log calls (default false) |
+| enableConsoleErrorToTrace       | Sets tracking error logs from loggers (console, bunyan, winston) as traces. If true errors will be returned as traces |
```

Comment:
It would be nice to mention current behavior that errors will generate Exception telemetry